types/jsonwebtoken/jsonwebtoken-tests.ts

```diff
@@ -4,8 +4,8 @@
  * Created by using code samples from https://github.com/auth0/node-jsonwebtoken.
  */
 
-import jwt = require("jsonwebtoken");
-import fs = require("fs");
+import jwt = require('jsonwebtoken');
```

Comment:
Your PR contains a load of style changes; would you consider only fixing the type issue that you have? I know that our repo contains a prettier config, but at the moment, we don't enforce it and I can't find the actual change in this PR with all of this other unrelated stuff.

Reply:
Issue addressed.